src/column/AVectorColumn.ts

```diff
@@ -95,11 +100,58 @@ export abstract class AVectorColumn<T, DATATYPE extends IVector<T, any>> extends
             d3.select(this).select('.vislist').style('display', 'none');
           });
         const m = new MultiForm(view, <HTMLElement>$multiformdivs.node(), this.multiFormParams($multiformdivs, domain));
-        m.addIconVisChooser(<HTMLElement>$header.node());
+        //assign visses
+        if(this.selectedAggVis){
+            VisManager.userSelectedAggregatedVisses[m.id.toString()] = this.selectedAggVis;
+         }
+        if(this.selectedUnaggVis){
+            VisManager.userSelectedUnaggregatedVisses[m.id.toString()] = this.selectedUnaggVis;
+        }
+        VisManager.setMultiformAggregationType(m.id.toString(), VisManager.aggregationType.UNAGGREGATED);
         this.multiformList.push(m);
+        const r = (<any>m).data.range;
+        let isSuccesor = Object.keys(idList).some((l, index) => {
+          let newRange = r.dims[0].asList().toString();
+          let originalRange = idList[l].dims[0].toString();
+          if (newRange === originalRange) {
+            VisManager.setMultiformAggregationType(m.id.toString(), VisManager.multiformAggregationType[l]);
+            return true;
+          } else {
+            let newRangeList = r.dims[0].asList().sort((a, b) => (a - b));
+            let oldRangeList = idList[l].dims[0].asList().sort((a, b) => (a - b));
+            if (this.superbag(oldRangeList, newRangeList)) {
+              VisManager.setMultiformAggregationType(m.id.toString(), VisManager.multiformAggregationType[l]);
+              return true;
+            }
+            if (this.superbag(newRangeList, oldRangeList)) {
+              VisManager.setMultiformAggregationType(m.id.toString(), VisManager.multiformAggregationType[l]);
+              return true;
+            }
+          }
+        });
+      });
+      Object.keys(idList).forEach((l) => {
+        delete VisManager.multiformAggregationType[l];
+        VisManager.removeUserVisses(l);
       });
     });
   }
+
+  private superbag(sup, sub) {
```

Comment:
Could you pleas shortly comment this function? From the name and code it's not clear what it does...